docs/reference/api/pages.md

```diff
@@ -373,6 +373,48 @@ The successful PATCH request returns the complete patched document. See the [pag
 
 The trash is part of the overall page tree in order to maintain the nesting structure. As such, there is not only a simple `trash` property to set `true`. Instead, set `_targetId` to `_trash` and `_position` to `lastChild` (or another position within the trash). You may similarly move pages out of the trash by moving them to a position relative to another page that is not in the trash.
 
+## `DELETE /api/v1/@apostrophecms/page/:_id`
+
+
+**Authentication required.**
+
+This API route **permanently deletes the page database document**. Moving pieces to the trash in the Apostrophe user interface or [using a PATCH request](#moving-pages-to-the-trash) do not permanently delete database documents and are typically better options.
```

Comment:
I don't think we need 'This API route'. Could leave it at 'Permanently' etc. I assume we're in an API route section already. I would also remove the part about 'typically better options'. We don't know that, we can trust their intent. I do think its useful to link to PATCH requests as you have.

Reply:
On the first point, this sentence is part of a normal paragraph (unlike the previous line), so I was trying to maintain normal grammar, hence the subject.